Makefile

```diff
@@ -1,9 +1,12 @@
 .PHONY: all clean
-.PHONY: apiserver ui
+.PHONY: apiserver ui dev
 
 DEP := $(shell command -v dep 2> /dev/null)
 STATIK := $(shell command -v statik 2> /dev/null)
 
+dev:
```

Comment:
Could you move this below the all target. The first target in the Makefile is the default, and we do not want the dev target as the default.